bin/kano-settings-onboot

```diff
@@ -139,21 +140,34 @@ def ensure_correct_browser(dry_run=False):
     browser = 'chromium-browser' if chromium_support else 'epiphany-browser'
 
     logger.info("Set browser to be: {}".format(browser))
-    if dry_run:
+    if not dry_run:
         run_cmd('update-alternatives --set {} /usr/bin/{}'.format(browser_link, browser))
 
 
-def ensure_correct_volume_levels():
+def ensure_correct_volume_levels(dry_run=False):
     """
-    Sets the default max_dB option in ALSA for CKC only.
+    Sets the default max_dB option in ALSA for CKCv1 only once.
     """
+    was_set_once = os.path.exists(CKC_V1_ONBOOT_VOLUME_FLAG)
+    if was_set_once:
+        return
+
     is_ckc = is_ck2_pro(with_dbus=False)
+    if not is_ckc:
+        return
```

Comment:
Do you want to also create the flag here so that we don't have to run the slow `is_ck2_pro()` at every boot for all kits other than CKCv1?

Reply:
Given that the average runtime of that function is between `0.0008` - `0.0016` I don't think it's worth the added complexity. If we're going for performance, we should think about splitting this entire script into parallel units, I think.